docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,197 @@
+.. _acs-public-coverage:
+ACSPublicCoverage
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSPublicCoverage dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset [2]_.
+Briefly, the UCI Adult dataset is commonly used as a benchmark dataset when comparing
+different algorithmic fairness interventions. While UCI Adult is used to predict 
+whether an individual's annual income is above $50,000, the authors created 
+ACSPublicCoverage to predict whether an individual is covered by public health insurance.
```

Comment:
Can you elaborate on what public health insurance means? Is that Medicare, Medicaid, or something else? If you find a good resource you can also link to that or cite it.

Reply:
Yes, public health insurance in the U.S. means either Medicare or Medicaid. People under age 65 may qualify for Medicare (which is normally reserved for people over age 65) if they have certain disabilities. I will elaborate on this in the documentation